IEEE-PSH/hackpsh-web
# Leaderboard Prototype

# Description
I made a leaderboard prototype for the leaderboard page. IUsed ShadCN's implementation of data table and removed unnecessary info. The table can be sorted by clicking on the column headers and filtered by using the search bar above it.
# Test Guidelines
- Navigate to 'leaderboard' page.
- Ensure table is sorted correctly.
- Ensure search filter works correctly.
# Notes
- Currently uses an array of data to test table.
- Currently uses mapping function to display data (not optimized).
- Behind master, merging conflicts expected.
